src/pages/courses/[course].tsx

```diff
@@ -24,11 +24,7 @@ export default function CoursePage({ openForm, content }: CoursePageProps) {
       <main sx={styles.CoursePageMain}>
         {/* @ts-ignore */}
         <GridLayout sx={styles.CoursePageGrid}>
-          <CourseBody
-            formatText={mockFormatText}
-            outlineText={mockOutlineText}
-            wrapperStyles={styles.CoursePageBody}
```

Comment:
might just want to double check removing these styles are okay, looks like it just has an `order` on it

Reply:
Ah good call, didn't mean to remove the styles.